week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -7,34 +7,54 @@ Their computer detects a list of nearby planets that have Oxygen in their atmosp
 To be safe, they need to land on the first unamed planet that has Oxygen levels between 19.5% and 23.5%.
 
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
+
+Some string methods that might help you here are .replace() and .substring(). Let's look at a quick
+example before trying the exercise.
 */
 
-function safeLevels() {
+/* .replace() allows us to add something where we removed something*/
+// let greeting = "Good Morning";
+// greeting.replace("Morning", "Evening"); // outputs Good Evening
+
+/* .substring() allows us to remove things from strings */
+// let dessert = "ice cream and pancakes";
+
+// let newdessert = dessert.substring(0, 9);
+
+// console.log(newdessert); // returns ice cream
 
+/* + + + + + + + */
+/* Now try the exercise */
+
+function safeLevels(oxygen) {
+  let safePlace = oxygen.find(function findOxygen(oxygenLevel) {
+    return oxygenLevel > "19.5%" && oxygenLevel < "23.5%";
```

Comment:
Excellent use of `.find` here Iman. The only thing I would suggest is that `oxygenLevel` is a string here still and you are comparing the string with a number. This works at the moment because JavaScript can compare the ASCII codes of strings. However, it might be worth you adding in an extra step to change `"24.2%" into a number first and then use it in your condition - it just makes it a little more secure. But hey is still works! Well done 😄